### DISCUSSION TITLE
getStaticProps not working

### DISCUSSION THREAD

**Discussion by noworry-b-still** [vercel/next.js#30664] [Help] [answered]
I wrapped the component with layout in the _app.js file,,. and in  my pages directory, i was using getStaticProps in a page but its not working. If i remove the llayout wrapper then it is wokring . what is  the issue here

**Comment by jamesmosier** [comment]
Do you have a code sample you could share that shows the issue?

**Comment by noworry-b-still** [comment, marked as answer]
Yeah thanks for replying. I just donno why it happened. But There was a
mentioning in documentation that we need to restart the server if we
modified our custom app (_app.ts)
it resolved the issue.
